src/os/bluestore/BlueStore.cc

```diff
@@ -30,6 +30,7 @@
 #include "include/stringify.h"
 #include "include/str_map.h"
 #include "include/util.h"
+#include "include/scope_guard.h"
```

Comment:
nit: You seem to add this to a wrong commit. I would expect to see this in "os/bluestore: use scope_guard to log latency". And it is probably worth mentioning in the log message conflicts section.

Reply:
> nit: You seem to add this to a wrong commit. I would expect to see this in "os/bluestore: use scope_guard to log latency". And it is probably worth mentioning in the log message conflicts section.

yep, fixed. Thanks!